src/config/index.js

```diff
@@ -4,4 +4,5 @@ export default {
   LMS_CLIENT_ID: process.env.LMS_CLIENT_ID,
   SECURE_COOKIES: process.env.NODE_ENV !== 'development',
   SEGMENT_KEY: process.env.SEGMENT_KEY,
+  NEW_RELIC_APP_ID: process.env.NEW_RELIC_APP_ID,
```

Comment:
I think we should parameterize the licenseKey and keep it in edx-internal, even if its the same for all of our apps just in case someone from the community grabs the code.